src/lib/stitching/vortex/__tests__/pricingContext.test.ts

```diff
@@ -138,9 +138,9 @@ describe("PricingContext type", () => {
     expect(JSON.parse(mockFetch.mock.calls[0][1].body).variables)
       .toMatchInlineSnapshot(`
       Object {
-        "_v1_artistId": "artist-id",
-        "_v2_category": "DRAWING_COLLAGE_OTHER_WORK_ON_PAPER",
-        "_v3_sizeScore": 225,
+        "_v0_artistId": "artist-id",
```

Comment:
Q1: this seems to me as an implementation detail and not important in practice. I didn't want to apply a patch to update it - what do you think?

Reply:
I'd agree, internal-ish argument names the lib uses for stitching queries doesn't seem like something that would affect anything if the mechanism changes.